index.html

```diff
@@ -7,24 +7,50 @@
   </head>
   <body>
     <!-- div for user menu -->
-    <div id="container-users">
+    <div id="container-users" class="">
       <h2>Teachers P.E.T.</h2>
       <div id="users">
         <nav id="nav-wrap" class="">
           <ul>
             <li a href="#" class="" id="menu-label">User</li>
-            <ul id="submenu">
-              <li class="user-btns" id="teacher">
-                <a href="#">Teacher</a></li>
-              <li class="user-btns" id="student">
-                <a href="#">Student</a></li>
-              <li class="user-btns" id ="parent">
-                <a href="#">Parent</a></li>
+            <ul class="submenu">
+              <a href="#">
+              <li class="menu-btns" id="teacher">
+                Teacher</li></a>
+                <a href="#">
+              <li class="menu-btns" id="student">
+                Student</li></a>
+                <a href="#">
+              <li class="menu-btns" id ="parent">
+                Parent</li></a>
             </ul>
           </ul>
         </nav>
       </div>
     </div>
+<!-- Container for teacher section -->
```

Comment:
You don't need a comment like this. The `div` has an `id` of `"teacher-container"`.